darwin/exporter/formats/coco.py

```diff
@@ -34,7 +34,9 @@ def export(annotation_files: Iterator[dt.AnnotationFile], output_dir: Path) -> N
     deprecated_in="0.7.7",
     removed_in="0.8.0",
     current_version=__version__,
-    details="Use the ``export`` instead.",
+    details="""This function is going to be turned into private. This means that breaking changes 
+    in its interface and implementation are to be expected. We encourage using ``export`` instead 
+    of calling this low-level function directly.""",
```

Comment:
Sorry for being a bit nitpicky, but couldn't we define this string once, so we can reuse it in all these places?